### ISSUE TITLE
Updates to /Clubs 

### ISSUE THREAD

**Issue by hannahkane** [mozilla/teach.mozilla.org#1545]
A few things to add:

1. The Clubs team has been working hard to finalize a series of resources for Club Captains. We need to make sure this content can be easily found by CCs. A longer term solution is to integrate this into an interactive Clubs Portal, but in the short term, we can add links to this content from https://teach.mozilla.org/Clubs

2. Add link to monthly office hours: http://mozilla.github.io/learning-networks/clubs/#officehours

3. Add brief explanation of difference between Mozilla Clubs and Firefox Clubs. @amirad or @kgorr - can you please provide short copy for this?

**Comment by amirad** [comment]
It's the list of guides/resources in this doc (at the end of 2015 reflection): https://docs.google.com/document/d/1JF56YrbVy42joIyaApW7h5lw-CFhPCma8EpEVo2I5_E/edit

**Comment by hannahkane** [comment]
Thanks, @amirad.

@sabrinang - this might be a good one to look at if you have time. The task would be to take a look at the guides in the doc and think about how we'd present links to them on /Clubs.

**Comment by hannahkane** [comment]
@sabrinang - I've added some content to this ticket. Can you take a look and let me know if you have any questions about it?

**Comment by KGorr** [comment]
@hannahkane, this is what I currently send out to folks who seem to need the extra explanation:

"Mozilla Club: Group of learners who together meet regularly to learn how to read, write and participate on the Web in a participatory way by making projects they care about, supported by a shared set of curricula

Firefox Club: Key grouping of Student Ambassadors at your school which function as Mozilla's liaison at these institutions and take part in fun activities that will help promote Mozilla, Firefox, and Mozilla's mission

In short, a Mozilla Club is more than just teaching about Firefox, but rather Web Literacy in general. While there are differences, the two types of clubs can overlap. You can read about one Club Captains thoughts on the overlap here: https://dunebuggie.wordpress.com/2015/07/30/firefox-student-ambassador-clubs-and-mozilla-clubs/ "

**Comment by hannahkane** [comment]
@KGorr - are there other commonly asked questions that we could group into a proper FAQ?

**Comment by KGorr** [comment]
@hannahkane - This doc may be useful. The Mozilla Clubs section starts on page 2. I update it any time there is a new question that I've not received before. https://docs.google.com/a/mozilla.com/document/d/18FUPmsHV30XpMPPMz-fY2FzLUT5UFdoQEPcOqYpmvU8/edit?usp=sharing

**Comment by sabrinang** [comment]
redpen draft here: https://redpen.io/p/he4d93fc5a7a4e6b0b

**Comment by KGorr** [comment]
Awesome job on the layout! It's beautiful. I made 2 small comments on redpen.

**Comment by hannahkane** [comment]
This looks great, @sabrinang. I added a couple comments, too.

@amirad - can you weigh in when you have some time?

**Comment by amirad** [comment]
Added some comments too!

**Comment by sabrinang** [comment]
**Updates:** https://redpen.io/p/he4d93fc5a7a4e6b0b

- removed the “what does a mozilla club look like in your…?” section since this info lives within the resources list we include below it
- removed “To get matched with a Regional Coordinator, please apply to be a Mozilla CC” text above apply button
- regarding moving Resources to another page, I agree that the accordion list could possibly get long as resources grow but we don’t have sub pages for secondary nav items and this is our interim solution (If we have it a subpage it won’t show up in our navigation as the current site is structured)
- included tab sections to showcase other localized resources *(3 other placeholder languages seen as examples in the mockup)* until we have a better localization solution site wide to handle localized content
- removed the word “guide” from all links except for the marketing your club section links.

**Comment by amirad** [comment]
Looking good Sabrina!

Right now we only have Portugese translated materials so have a bunch that we can place there but we might have more in the future so good to plan for that. 

The page does feel super long with lots of info. @hannahkane with the new crm for guides that will live on it's own page right?

**Comment by sabrinang** [comment]
@amirad Thanks for feedback! Can you supply the Portugese translated materials links? (will be useful to have ready when we hand off to devs)

**Updates:** https://redpen.io/thcaaef4b84ba1a04c
- removed last paragraph on being matched with an RC
- shows Portugese tab
- removed unnecessary dividers on FAQ section
- aligning with list style on https://teach.mozilla.org/events/resources/ to vertical space on the titles and doesn't collapse (Even though it appears long all the links can be displayed at once)

This page could get quite lengthy depending on expansion of links in the future but to my knowledge this is a interim solution to make updates to the page and in the future these resources can be housed on the cms that is in development but will leave that questions for @hannahkane

**Comment by hannahkane** [comment]
fact! this is an interim solution that I think works for now.

I think this looks good. If design is approved, I think we can close this and move to implementation.

**Comment by sabrinang** [comment]
design review request for https://redpen.io/p/he4d93fc5a7a4e6b0b
R? @cassiemc @kristinashu

**Comment by kristinashu** [comment]
@sabrinang looks great! I just made a comments in redpen about disabling the map scroll-to-zoom on mobile.

**Comment by amirad** [comment]
Sure @sabrinang they are all listed on this page: http://leaomozillaclubs.github.io/leadership_meetings_2016.html though I'm not able to access some of them so will send Geraldo a message right now to see what's happening

**Comment by amirad** [comment]
Here's the updated link with all the working guides @sabrinang http://leaomozillaclubs.com/leadership_meetings_2016.html

**Comment by sabrinang** [comment]
Thanks @amirad for the links and I included it in a redpen here (to show how localized materials look when selected): https://redpen.io/kn68b61855dcda9ff9

Thanks for the review @kristinashu and @cassiemc. I made a separate ticket regarding the map #1671 *(included it in design audit = not a blocker)*

**Comment by amirad** [comment]
Looks good!

**Comment by sabrinang** [comment]
Closing this ticket and implementation ticket is #1605

**Comment by hannahkane** [comment]
@amirad - Wondering if we should add a link below the table of resources to the Clubs folder of the Learning-Networks repo with copy like, ""We add new resources when we can. To see the latest, click here." This way we'd get the benefit of Sabrina's pretty table design, but would also be pointing people towards an updated set of resources. What do you think?

**Comment by amirad** [comment]
@hannahkane as discussed, linking to the repo might be confusing for visitors so we won't do it!

**Comment by hannahkane** [comment]
yeah, cool. Let's not confuse ppl! OK, the implementation ticket is in the queue for this heartbeat, but is a P2.
